SD-K9/src/app/pages/home/home.page.ts

```diff
@@ -60,6 +68,18 @@ export class HomePage implements OnInit {
     });
   }
 
+  getLocation(){
+    this._unsubscribe = this._buildingInfoStore.subscribe(() => {
+      this.location = this._buildingInfoStore.getBuildingInfo().building;
+      if(this.startLocation==null){
+        this.startLocation=this.location;
+      }
+      else{
+        this.endLocation=this.location;
```

Comment:
Here, since the master is using DirectionFormState, you will need to change it to `this._directionFormState.getDirectionFormState( )`